Whitelist: Update Information, Clarify, & Add Coloring To Table

RoboticMind opened the pull request:
These are minor, but just helps make it easier to find a project on the page and clearer about the greylist monitoring website since I have seen some people get confused by it in the past

barton2526 commented:
Can we fix this line while we're at it?

`The whitelist is verifiable within the Windows client (coming to Linux soon).`

RoboticMind commented:
Did that and added a link to the whitelist process doc as well where it talks about wanting a project to be whitelisted. Also added some background colouring for the whitelist table so its a bit easier to tell at a glance if a project has GPU projects or CPU projects

![image](https://user-images.githubusercontent.com/30132912/100101936-e29a4100-2e30-11eb-93ef-f1603fffb1a4.png)
